fair/emissions_driven.py

```diff
@@ -10,7 +10,7 @@ def run(inp_df, cfg):
     Parameters
     ----------
     inp_df : :obj:`pd.DataFrame`
-        Input :obj:`pd.DataFrame` containing the timeseries to run
+        Input :obj:`pd.DataFrame` containing the timeseries to run, in IAMC compliant DataFrame format (i.e. A multiIndex of Model, Region, Scenario, Unit, Variable then Columns for time)
```

Comment:
Is the aim for it to be IAMC compliant or simply useable by https://github.com/IAMconsortium/pyam?

Reply:
I figured I might as well make the output IAMC compliant, but you're right that actually it makes rather more sense to have input as useable rather than explicitly compliant (but I'm guessing the code doesn't need to change at all